src/components/com_tjcertificate/administrator/libraries/certificate.php

```diff
@@ -478,6 +478,8 @@ public function save()
 				$table->issued_on = Factory::getDate()->toSql();
 			}
 
+			$table->created_by = !empty($this->created_by) ? $this->created_by : 0;
```

Comment:
@MeghaBiranje If created by is not present and record is being created then by default the value should be user is of logged in user